cliquet/views/errors.py

```diff
@@ -82,6 +82,10 @@ def error(context, request):
         logger.critical(context.original, exc_info=True)
         return service_unavailable(context, request)
 
+    if hasattr(request, 'raven'):
+        ident = request.raven.get_ident(request.raven.captureException())
+        print("Exception captured in event number: %s" % ident)
```

Comment:
If a DB constraint is violated, it will be catched in the backend and a BackendError will be raised. And sentry will not be aware of it, because it is returned 4 LOC above